internal/http_cache/http_cache.go

```diff
@@ -102,7 +102,7 @@ func downloadCache(w http.ResponseWriter, r *http.Request, cacheKey string) {
 			in, err := cacheStream.Recv()
 			if in != nil && in.RedirectUrl != "" {
```

Comment:
Since we don't do redirects anymore it would be nice to rename variables like this for clarity.